Description: Upstream changes introduced in version 0.2.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 gstlal-ugly (0.2.1-1) unstable; urgency=low
 .
   * New upstream release, fixes nds2 dependency problem
 .
 The person named in the Author field signed this changelog entry.
Author: Carsten Aulbert <carsten.aulbert@aei.mpg.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ gstlal-ugly-0.2.1/config.log
@@ -0,0 +1,1259 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by gstlal-ugly configure 0.2.1, which was
+generated by GNU Autoconf 2.68.  Invocation command line was
+
+  $ ./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=${prefix}/include --mandir=${prefix}/share/man --infodir=${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --libexecdir=${prefix}/lib/gstlal-ugly --disable-maintainer-mode --disable-dependency-tracking
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = bob
+uname -m = x86_64
+uname -r = 3.2.1-atlas
+uname -s = Linux
+uname -v = #1 SMP Mon Jan 16 13:29:43 CET 2012
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /opt/lscsoft/gst/bin
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+PATH: /usr/bin/X11
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2431: checking for a BSD-compatible install
+configure:2499: result: /usr/bin/install -c
+configure:2510: checking whether build environment is sane
+configure:2560: result: yes
+configure:2701: checking for a thread-safe mkdir -p
+configure:2740: result: /bin/mkdir -p
+configure:2753: checking for gawk
+configure:2783: result: no
+configure:2753: checking for mawk
+configure:2769: found /usr/bin/mawk
+configure:2780: result: mawk
+configure:2791: checking whether make sets $(MAKE)
+configure:2813: result: yes
+configure:2888: checking how to create a ustar tar archive
+configure:2901: tar --version
+tar (GNU tar) 1.23
+Copyright (C) 2010 Free Software Foundation, Inc.
+License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+
+Written by John Gilmore and Jay Fenlason.
+configure:2904: $? = 0
+configure:2944: tardir=conftest.dir && eval tar --format=ustar -chf - "$tardir" >conftest.tar
+configure:2947: $? = 0
+configure:2951: tar -xf - <conftest.tar
+configure:2954: $? = 0
+configure:2967: result: gnutar
+configure:3036: checking for style of include used by make
+configure:3064: result: GNU
+configure:3135: checking for gcc
+configure:3151: found /usr/bin/gcc
+configure:3162: result: gcc
+configure:3391: checking for C compiler version
+configure:3400: gcc --version >&5
+gcc (Debian 4.4.5-8) 4.4.5
+Copyright (C) 2010 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3411: $? = 0
+configure:3400: gcc -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.5-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.4 --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.5 (Debian 4.4.5-8) 
+configure:3411: $? = 0
+configure:3400: gcc -V >&5
+gcc: '-V' option must have argument
+configure:3411: $? = 1
+configure:3400: gcc -qversion >&5
+gcc: unrecognized option '-qversion'
+gcc: no input files
+configure:3411: $? = 1
+configure:3431: checking whether the C compiler works
+configure:3453: gcc -g -O2   conftest.c  >&5
+configure:3457: $? = 0
+configure:3505: result: yes
+configure:3508: checking for C compiler default output file name
+configure:3510: result: a.out
+configure:3516: checking for suffix of executables
+configure:3523: gcc -o conftest -g -O2   conftest.c  >&5
+configure:3527: $? = 0
+configure:3549: result: 
+configure:3571: checking whether we are cross compiling
+configure:3579: gcc -o conftest -g -O2   conftest.c  >&5
+configure:3583: $? = 0
+configure:3590: ./conftest
+configure:3594: $? = 0
+configure:3609: result: no
+configure:3614: checking for suffix of object files
+configure:3636: gcc -c -g -O2  conftest.c >&5
+configure:3640: $? = 0
+configure:3661: result: o
+configure:3665: checking whether we are using the GNU C compiler
+configure:3684: gcc -c -g -O2  conftest.c >&5
+configure:3684: $? = 0
+configure:3693: result: yes
+configure:3702: checking whether gcc accepts -g
+configure:3722: gcc -c -g  conftest.c >&5
+configure:3722: $? = 0
+configure:3763: result: yes
+configure:3780: checking for gcc option to accept ISO C89
+configure:3844: gcc  -c -g -O2  conftest.c >&5
+configure:3844: $? = 0
+configure:3857: result: none needed
+configure:3879: checking dependency style of gcc
+configure:3990: result: none
+configure:4005: checking for gcc option to accept ISO C99
+configure:4154: gcc  -c -g -O2  conftest.c >&5
+conftest.c:61: error: expected ';', ',' or ')' before 'text'
+conftest.c: In function 'main':
+conftest.c:115: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'newvar'
+conftest.c:115: error: 'newvar' undeclared (first use in this function)
+conftest.c:115: error: (Each undeclared identifier is reported only once
+conftest.c:115: error: for each function it appears in.)
+conftest.c:125: error: 'for' loop initial declarations are only allowed in C99 mode
+conftest.c:125: note: use option -std=c99 or -std=gnu99 to compile your code
+configure:4154: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "gstlal-ugly"
+| #define PACKAGE_TARNAME "gstlal-ugly"
+| #define PACKAGE_VERSION "0.2.1"
+| #define PACKAGE_STRING "gstlal-ugly 0.2.1"
+| #define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+| #define PACKAGE_URL ""
+| #define PACKAGE "gstlal-ugly"
+| #define VERSION "0.2.1"
+| /* end confdefs.h.  */
+| #include <stdarg.h>
+| #include <stdbool.h>
+| #include <stdlib.h>
+| #include <wchar.h>
+| #include <stdio.h>
+| 
+| // Check varargs macros.  These examples are taken from C99 6.10.3.5.
+| #define debug(...) fprintf (stderr, __VA_ARGS__)
+| #define showlist(...) puts (#__VA_ARGS__)
+| #define report(test,...) ((test) ? puts (#test) : printf (__VA_ARGS__))
+| static void
+| test_varargs_macros (void)
+| {
+|   int x = 1234;
+|   int y = 5678;
+|   debug ("Flag");
+|   debug ("X = %d\n", x);
+|   showlist (The first, second, and third items.);
+|   report (x>y, "x is %d but y is %d", x, y);
+| }
+| 
+| // Check long long types.
+| #define BIG64 18446744073709551615ull
+| #define BIG32 4294967295ul
+| #define BIG_OK (BIG64 / BIG32 == 4294967297ull && BIG64 % BIG32 == 0)
+| #if !BIG_OK
+|   your preprocessor is broken;
+| #endif
+| #if BIG_OK
+| #else
+|   your preprocessor is broken;
+| #endif
+| static long long int bignum = -9223372036854775807LL;
+| static unsigned long long int ubignum = BIG64;
+| 
+| struct incomplete_array
+| {
+|   int datasize;
+|   double data[];
+| };
+| 
+| struct named_init {
+|   int number;
+|   const wchar_t *name;
+|   double average;
+| };
+| 
+| typedef const char *ccp;
+| 
+| static inline int
+| test_restrict (ccp restrict text)
+| {
+|   // See if C++-style comments work.
+|   // Iterate through items via the restricted pointer.
+|   // Also check for declarations in for loops.
+|   for (unsigned int i = 0; *(text+i) != '\0'; ++i)
+|     continue;
+|   return 0;
+| }
+| 
+| // Check varargs and va_copy.
+| static void
+| test_varargs (const char *format, ...)
+| {
+|   va_list args;
+|   va_start (args, format);
+|   va_list args_copy;
+|   va_copy (args_copy, args);
+| 
+|   const char *str;
+|   int number;
+|   float fnumber;
+| 
+|   while (*format)
+|     {
+|       switch (*format++)
+| 	{
+| 	case 's': // string
+| 	  str = va_arg (args_copy, const char *);
+| 	  break;
+| 	case 'd': // int
+| 	  number = va_arg (args_copy, int);
+| 	  break;
+| 	case 'f': // float
+| 	  fnumber = va_arg (args_copy, double);
+| 	  break;
+| 	default:
+| 	  break;
+| 	}
+|     }
+|   va_end (args_copy);
+|   va_end (args);
+| }
+| 
+| int
+| main ()
+| {
+| 
+|   // Check bool.
+|   _Bool success = false;
+| 
+|   // Check restrict.
+|   if (test_restrict ("String literal") == 0)
+|     success = true;
+|   char *restrict newvar = "Another string";
+| 
+|   // Check varargs.
+|   test_varargs ("s, d' f .", "string", 65, 34.234);
+|   test_varargs_macros ();
+| 
+|   // Check flexible array members.
+|   struct incomplete_array *ia =
+|     malloc (sizeof (struct incomplete_array) + (sizeof (double) * 10));
+|   ia->datasize = 10;
+|   for (int i = 0; i < ia->datasize; ++i)
+|     ia->data[i] = i * 1.234;
+| 
+|   // Check named initializers.
+|   struct named_init ni = {
+|     .number = 34,
+|     .name = L"Test wide string",
+|     .average = 543.34343,
+|   };
+| 
+|   ni.number = 58;
+| 
+|   int dynamic_array[ni.number];
+|   dynamic_array[ni.number - 1] = 543;
+| 
+|   // work around unused variable warnings
+|   return (!success || bignum == 0LL || ubignum == 0uLL || newvar[0] == 'x'
+| 	  || dynamic_array[ni.number - 1] != 543);
+| 
+|   ;
+|   return 0;
+| }
+configure:4154: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:4154: $? = 0
+configure:4174: result: -std=gnu99
+configure:4241: checking for g++
+configure:4257: found /usr/bin/g++
+configure:4268: result: g++
+configure:4295: checking for C++ compiler version
+configure:4304: g++ --version >&5
+g++ (Debian 4.4.5-8) 4.4.5
+Copyright (C) 2010 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:4315: $? = 0
+configure:4304: g++ -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.5-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.4 --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.5 (Debian 4.4.5-8) 
+configure:4315: $? = 0
+configure:4304: g++ -V >&5
+g++: '-V' option must have argument
+configure:4315: $? = 1
+configure:4304: g++ -qversion >&5
+g++: unrecognized option '-qversion'
+g++: no input files
+configure:4315: $? = 1
+configure:4319: checking whether we are using the GNU C++ compiler
+configure:4338: g++ -c -g -O2  conftest.cpp >&5
+configure:4338: $? = 0
+configure:4347: result: yes
+configure:4356: checking whether g++ accepts -g
+configure:4376: g++ -c -g  conftest.cpp >&5
+configure:4376: $? = 0
+configure:4417: result: yes
+configure:4442: checking dependency style of g++
+configure:4553: result: none
+configure:4605: checking build system type
+configure:4619: result: x86_64-pc-linux-gnu
+configure:4639: checking host system type
+configure:4652: result: x86_64-pc-linux-gnu
+configure:4693: checking how to print strings
+configure:4720: result: printf
+configure:4741: checking for a sed that does not truncate output
+configure:4805: result: /bin/sed
+configure:4823: checking for grep that handles long lines and -e
+configure:4881: result: /bin/grep
+configure:4886: checking for egrep
+configure:4948: result: /bin/grep -E
+configure:4953: checking for fgrep
+configure:5015: result: /bin/grep -F
+configure:5050: checking for ld used by gcc -std=gnu99
+configure:5117: result: /usr/bin/ld
+configure:5124: checking if the linker (/usr/bin/ld) is GNU ld
+configure:5139: result: yes
+configure:5151: checking for BSD- or MS-compatible name lister (nm)
+configure:5200: result: /usr/bin/nm -B
+configure:5330: checking the name lister (/usr/bin/nm -B) interface
+configure:5337: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:5340: /usr/bin/nm -B "conftest.o"
+configure:5343: output
+0000000000000000 B some_variable
+configure:5350: result: BSD nm
+configure:5353: checking whether ln -s works
+configure:5357: result: yes
+configure:5365: checking the maximum length of command line arguments
+configure:5490: result: 3458764513820540925
+configure:5507: checking whether the shell understands some XSI constructs
+configure:5517: result: yes
+configure:5521: checking whether the shell understands "+="
+configure:5527: result: yes
+configure:5562: checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format
+configure:5602: result: func_convert_file_noop
+configure:5609: checking how to convert x86_64-pc-linux-gnu file names to toolchain format
+configure:5629: result: func_convert_file_noop
+configure:5636: checking for /usr/bin/ld option to reload object files
+configure:5643: result: -r
+configure:5717: checking for objdump
+configure:5733: found /usr/bin/objdump
+configure:5744: result: objdump
+configure:5776: checking how to recognize dependent libraries
+configure:5978: result: pass_all
+configure:6063: checking for dlltool
+configure:6093: result: no
+configure:6123: checking how to associate runtime and link libraries
+configure:6150: result: printf %s\n
+configure:6211: checking for ar
+configure:6227: found /usr/bin/ar
+configure:6238: result: ar
+configure:6275: checking for archiver @FILE support
+configure:6292: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:6292: $? = 0
+configure:6295: ar cru libconftest.a @conftest.lst >&5
+configure:6298: $? = 0
+configure:6303: ar cru libconftest.a @conftest.lst >&5
+ar: conftest.o: No such file or directory
+configure:6306: $? = 1
+configure:6318: result: @
+configure:6376: checking for strip
+configure:6392: found /usr/bin/strip
+configure:6403: result: strip
+configure:6475: checking for ranlib
+configure:6491: found /usr/bin/ranlib
+configure:6502: result: ranlib
+configure:6604: checking command to parse /usr/bin/nm -B output from gcc -std=gnu99 object
+configure:6723: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:6726: $? = 0
+configure:6730: /usr/bin/nm -B conftest.o \| sed -n -e 's/^.*[ ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[ ][ ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p' | sed '/ __gnu_lto/d' \> conftest.nm
+configure:6733: $? = 0
+configure:6799: gcc -std=gnu99 -o conftest -g -O2   conftest.c conftstm.o >&5
+configure:6802: $? = 0
+configure:6840: result: ok
+configure:6877: checking for sysroot
+configure:6907: result: no
+configure:6984: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:6987: $? = 0
+configure:7150: checking for mt
+configure:7180: result: no
+configure:7200: checking if : is a manifest tool
+configure:7206: : '-?'
+configure:7214: result: no
+configure:7846: checking how to run the C preprocessor
+configure:7877: gcc -std=gnu99 -E  conftest.c
+configure:7877: $? = 0
+configure:7891: gcc -std=gnu99 -E  conftest.c
+conftest.c:11:28: error: ac_nonexistent.h: No such file or directory
+configure:7891: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "gstlal-ugly"
+| #define PACKAGE_TARNAME "gstlal-ugly"
+| #define PACKAGE_VERSION "0.2.1"
+| #define PACKAGE_STRING "gstlal-ugly 0.2.1"
+| #define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+| #define PACKAGE_URL ""
+| #define PACKAGE "gstlal-ugly"
+| #define VERSION "0.2.1"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:7916: result: gcc -std=gnu99 -E
+configure:7936: gcc -std=gnu99 -E  conftest.c
+configure:7936: $? = 0
+configure:7950: gcc -std=gnu99 -E  conftest.c
+conftest.c:11:28: error: ac_nonexistent.h: No such file or directory
+configure:7950: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "gstlal-ugly"
+| #define PACKAGE_TARNAME "gstlal-ugly"
+| #define PACKAGE_VERSION "0.2.1"
+| #define PACKAGE_STRING "gstlal-ugly 0.2.1"
+| #define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+| #define PACKAGE_URL ""
+| #define PACKAGE "gstlal-ugly"
+| #define VERSION "0.2.1"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:7979: checking for ANSI C header files
+configure:7999: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:7999: $? = 0
+configure:8072: gcc -std=gnu99 -o conftest -g -O2   conftest.c  >&5
+configure:8072: $? = 0
+configure:8072: ./conftest
+configure:8072: $? = 0
+configure:8083: result: yes
+configure:8096: checking for sys/types.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for sys/stat.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for stdlib.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for string.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for memory.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for strings.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for inttypes.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for stdint.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8096: checking for unistd.h
+configure:8096: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8096: $? = 0
+configure:8096: result: yes
+configure:8110: checking for dlfcn.h
+configure:8110: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:8110: $? = 0
+configure:8110: result: yes
+configure:8308: checking for objdir
+configure:8323: result: .libs
+configure:8594: checking if gcc -std=gnu99 supports -fno-rtti -fno-exceptions
+configure:8612: gcc -std=gnu99 -c -g -O2  -fno-rtti -fno-exceptions conftest.c >&5
+cc1: warning: command line option "-fno-rtti" is valid for C++/ObjC++ but not for C
+configure:8616: $? = 0
+configure:8629: result: no
+configure:8939: checking for gcc -std=gnu99 option to produce PIC
+configure:8946: result: -fPIC -DPIC
+configure:8954: checking if gcc -std=gnu99 PIC flag -fPIC -DPIC works
+configure:8972: gcc -std=gnu99 -c -g -O2  -fPIC -DPIC -DPIC conftest.c >&5
+configure:8976: $? = 0
+configure:8989: result: yes
+configure:9018: checking if gcc -std=gnu99 static flag -static works
+configure:9046: result: yes
+configure:9061: checking if gcc -std=gnu99 supports -c -o file.o
+configure:9082: gcc -std=gnu99 -c -g -O2  -o out/conftest2.o conftest.c >&5
+configure:9086: $? = 0
+configure:9108: result: yes
+configure:9116: checking if gcc -std=gnu99 supports -c -o file.o
+configure:9163: result: yes
+configure:9196: checking whether the gcc -std=gnu99 linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
+configure:10354: result: yes
+configure:10391: checking whether -lc should be explicitly linked in
+configure:10399: gcc -std=gnu99 -c -g -O2  conftest.c >&5
+configure:10402: $? = 0
+configure:10417: gcc -std=gnu99 -shared  -fPIC -DPIC conftest.o  -v -Wl,-soname -Wl,conftest -o conftest 2\>\&1 \| /bin/grep  -lc  \>/dev/null 2\>\&1
+configure:10420: $? = 0
+configure:10434: result: no
+configure:10599: checking dynamic linker characteristics
+configure:11113: gcc -std=gnu99 -o conftest -g -O2   -Wl,-rpath -Wl,/foo conftest.c  >&5
+configure:11113: $? = 0
+configure:11335: result: GNU/Linux ld.so
+configure:11442: checking how to hardcode library paths into programs
+configure:11467: result: immediate
+configure:12007: checking whether stripping libraries is possible
+configure:12012: result: yes
+configure:12047: checking if libtool supports shared libraries
+configure:12049: result: yes
+configure:12052: checking whether to build shared libraries
+configure:12073: result: yes
+configure:12076: checking whether to build static libraries
+configure:12080: result: yes
+configure:12103: checking how to run the C++ preprocessor
+configure:12130: g++ -E  conftest.cpp
+configure:12130: $? = 0
+configure:12144: g++ -E  conftest.cpp
+conftest.cpp:23:28: error: ac_nonexistent.h: No such file or directory
+configure:12144: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "gstlal-ugly"
+| #define PACKAGE_TARNAME "gstlal-ugly"
+| #define PACKAGE_VERSION "0.2.1"
+| #define PACKAGE_STRING "gstlal-ugly 0.2.1"
+| #define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+| #define PACKAGE_URL ""
+| #define PACKAGE "gstlal-ugly"
+| #define VERSION "0.2.1"
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:12169: result: g++ -E
+configure:12189: g++ -E  conftest.cpp
+configure:12189: $? = 0
+configure:12203: g++ -E  conftest.cpp
+conftest.cpp:23:28: error: ac_nonexistent.h: No such file or directory
+configure:12203: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "gstlal-ugly"
+| #define PACKAGE_TARNAME "gstlal-ugly"
+| #define PACKAGE_VERSION "0.2.1"
+| #define PACKAGE_STRING "gstlal-ugly 0.2.1"
+| #define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+| #define PACKAGE_URL ""
+| #define PACKAGE "gstlal-ugly"
+| #define VERSION "0.2.1"
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:12373: checking for ld used by g++
+configure:12440: result: /usr/bin/ld -m elf_x86_64
+configure:12447: checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld
+configure:12462: result: yes
+configure:12517: checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
+configure:13521: result: yes
+configure:13556: g++ -c -g -O2  conftest.cpp >&5
+configure:13559: $? = 0
+configure:14079: checking for g++ option to produce PIC
+configure:14086: result: -fPIC -DPIC
+configure:14094: checking if g++ PIC flag -fPIC -DPIC works
+configure:14112: g++ -c -g -O2  -fPIC -DPIC -DPIC conftest.cpp >&5
+configure:14116: $? = 0
+configure:14129: result: yes
+configure:14152: checking if g++ static flag -static works
+configure:14180: result: yes
+configure:14192: checking if g++ supports -c -o file.o
+configure:14213: g++ -c -g -O2  -o out/conftest2.o conftest.cpp >&5
+configure:14217: $? = 0
+configure:14239: result: yes
+configure:14244: checking if g++ supports -c -o file.o
+configure:14291: result: yes
+configure:14321: checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
+configure:14355: result: yes
+configure:14498: checking dynamic linker characteristics
+configure:15168: result: GNU/Linux ld.so
+configure:15221: checking how to hardcode library paths into programs
+configure:15246: result: immediate
+configure:15359: checking for pkg-config
+configure:15377: found /usr/bin/pkg-config
+configure:15389: result: /usr/bin/pkg-config
+configure:15414: checking pkg-config is at least version 0.9.0
+configure:15417: result: yes
+configure:15465: checking for a Python interpreter with version >= 2.6
+configure:15482: python -c import sys # split strings by '.' and convert to numeric. Append some zeros # because we need at least 4 digits for the hex conversion. # map returns an iterator in Python 3.0 and a list in 2.x minver = list(map(int, '2.6'.split('.'))) + [0, 0, 0] minverhex = 0 # xrange is not present in Python 3.0 and range returns an iterator for i in list(range(0, 4)): minverhex = (minverhex << 8) + minver[i] sys.exit(sys.hexversion < minverhex)
+configure:15485: $? = 0
+configure:15491: result: python
+configure:15499: checking for python
+configure:15517: found /usr/bin/python
+configure:15529: result: /usr/bin/python
+configure:15547: checking for python version
+configure:15554: result: 2.6
+configure:15566: checking for python platform
+configure:15573: result: linux2
+configure:15580: checking for python script directory
+configure:15608: result: ${prefix}/lib/python2.6/dist-packages
+configure:15617: checking for python extension module directory
+configure:15645: result: ${exec_prefix}/lib/python2.6/dist-packages
+configure:15666: checking for python2.6
+configure:15696: result: /usr/bin/python
+configure:15712: checking for a version of Python >= '2.1.0'
+configure:15738: result: yes
+configure:15769: checking for the distutils Python package
+configure:15773: result: yes
+configure:15787: checking for Python include path
+configure:15797: result: -I/usr/include/python2.6
+configure:15804: checking for Python library path
+configure:15889: result: -L/usr/lib -lpython2.6
+configure:15896: checking for Python site-packages path
+configure:15902: result: /usr/lib/python2.6/dist-packages
+configure:15909: checking python extra libraries
+configure:15916: result: -lssl -lcrypto  -lssl -lcrypto      -L/usr/lib -lz -lpthread -ldl  -lutil
+configure:15923: checking python extra linking flags
+configure:15930: result: -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions
+configure:15937: checking consistency of all components of python development environment
+configure:15963: gcc -std=gnu99 -o conftest -g -O2  -I/usr/include/python2.6  conftest.c  -L/usr/lib -lpython2.6 -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions -lssl -lcrypto  -lssl -lcrypto      -L/usr/lib -lz -lpthread -ldl  -lutil >&5
+configure:15963: $? = 0
+configure:15980: result: yes
+configure:16023: checking AM_CFLAGS for maximum warnings
+configure:16050: gcc -std=gnu99 -c -g -O2 -pedantic  -Wall  conftest.c >&5
+configure:16050: $? = 0
+configure:16064: result: -Wall
+configure:16077: : AM_CFLAGS="$AM_CFLAGS $ac_cv_cflags_warn_all"
+configure:16080: $? = 0
+configure:16106: checking for main in -lm
+configure:16125: gcc -std=gnu99 -o conftest -g -O2   conftest.c -lm   >&5
+configure:16125: $? = 0
+configure:16134: result: yes
+configure:16156: checking for FFTW
+configure:16163: $PKG_CONFIG --exists --print-errors "fftw3"
+configure:16166: $? = 0
+configure:16180: $PKG_CONFIG --exists --print-errors "fftw3"
+configure:16183: $? = 0
+configure:16241: result: yes
+configure:16269: checking for NDS
+configure:16276: $PKG_CONFIG --exists --print-errors "nds2-client >= 0.9.0c"
+Package nds2-client was not found in the pkg-config search path.
+Perhaps you should add the directory containing `nds2-client.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'nds2-client' found
+configure:16279: $? = 1
+configure:16293: $PKG_CONFIG --exists --print-errors "nds2-client >= 0.9.0c"
+Package nds2-client was not found in the pkg-config search path.
+Perhaps you should add the directory containing `nds2-client.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'nds2-client' found
+configure:16296: $? = 1
+configure:16310: result: no
+No package 'nds2-client' found
+configure:16330: WARNING: Not found!
+configure:16388: checking for cairo
+configure:16395: $PKG_CONFIG --exists --print-errors "cairo"
+Package cairo was not found in the pkg-config search path.
+Perhaps you should add the directory containing `cairo.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'cairo' found
+configure:16398: $? = 1
+configure:16412: $PKG_CONFIG --exists --print-errors "cairo"
+Package cairo was not found in the pkg-config search path.
+Perhaps you should add the directory containing `cairo.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'cairo' found
+configure:16415: $? = 1
+configure:16429: result: no
+No package 'cairo' found
+configure:16449: WARNING: Not found!
+configure:16501: checking for gstreamer
+configure:16508: $PKG_CONFIG --exists --print-errors "gstreamer-${GSTREAMER_RELEASE} >= ${MIN_GSTREAMER_VERSION} gstreamer-base-${GSTREAMER_RELEASE} >= ${MIN_GSTREAMER_VERSION}"
+configure:16511: $? = 0
+configure:16525: $PKG_CONFIG --exists --print-errors "gstreamer-${GSTREAMER_RELEASE} >= ${MIN_GSTREAMER_VERSION} gstreamer-base-${GSTREAMER_RELEASE} >= ${MIN_GSTREAMER_VERSION}"
+configure:16528: $? = 0
+configure:16586: result: yes
+configure:16598: If you have chosen to install the software in a location not included in your gstreamer's default search path, you might need to add the directory
+
+	${exec_prefix}/lib/gstreamer-0.10
+
+to your GST_PLUGIN_PATH environment variable.
+configure:16621: gcc -std=gnu99 -c -pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2    conftest.c >&5
+conftest.c:26:35: error: gst/base/gstbaseparse.h: No such file or directory
+configure:16621: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "gstlal-ugly"
+| #define PACKAGE_TARNAME "gstlal-ugly"
+| #define PACKAGE_VERSION "0.2.1"
+| #define PACKAGE_STRING "gstlal-ugly 0.2.1"
+| #define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+| #define PACKAGE_URL ""
+| #define PACKAGE "gstlal-ugly"
+| #define VERSION "0.2.1"
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| #define HAVE_PYTHON "2.6"
+| #define HAVE_LIBM 1
+| #define GSTLAL_FFTW_WISDOM_ENV "GSTLAL_FFTW_WISDOM"
+| /* end confdefs.h.  */
+| #include <gst/base/gstbaseparse.h>
+| int
+| main ()
+| {
+| 
+|   ;
+|   return 0;
+| }
+configure:16653: checking for gstreamer_video
+configure:16660: $PKG_CONFIG --exists --print-errors "gstreamer-video-${GSTREAMER_VIDEO_RELEASE} >= ${MIN_GSTREAMER_VIDEO_VERSION}"
+configure:16663: $? = 0
+configure:16677: $PKG_CONFIG --exists --print-errors "gstreamer-video-${GSTREAMER_VIDEO_RELEASE} >= ${MIN_GSTREAMER_VIDEO_VERSION}"
+configure:16680: $? = 0
+configure:16738: result: yes
+configure:16757: checking for gstreamer_audio
+configure:16764: $PKG_CONFIG --exists --print-errors "gstreamer-audio-${GSTREAMER_VIDEO_RELEASE} >= ${MIN_GSTREAMER_VIDEO_VERSION}"
+configure:16767: $? = 0
+configure:16781: $PKG_CONFIG --exists --print-errors "gstreamer-audio-${GSTREAMER_VIDEO_RELEASE} >= ${MIN_GSTREAMER_VIDEO_VERSION}"
+configure:16784: $? = 0
+configure:16842: result: yes
+configure:16857: checking for pygobject
+configure:16864: $PKG_CONFIG --exists --print-errors "pygobject-2.0"
+configure:16867: $? = 0
+configure:16881: $PKG_CONFIG --exists --print-errors "pygobject-2.0"
+configure:16884: $? = 0
+configure:16942: result: yes
+configure:16961: checking python module: numpy
+configure:16966: result: yes
+configure:16984: checking for numpy/arrayobject.h
+configure:16984: gcc -std=gnu99 -c -g -O2 -I/usr/include/python2.6 -I/usr/lib/pymodules/python2.6/numpy/core/include  conftest.c >&5
+configure:16984: $? = 0
+configure:16984: result: yes
+configure:17013: checking for framecpp
+configure:17020: $PKG_CONFIG --exists --print-errors "framecpp >= 1.19.16"
+Package framecpp was not found in the pkg-config search path.
+Perhaps you should add the directory containing `framecpp.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'framecpp' found
+configure:17023: $? = 1
+configure:17037: $PKG_CONFIG --exists --print-errors "framecpp >= 1.19.16"
+Package framecpp was not found in the pkg-config search path.
+Perhaps you should add the directory containing `framecpp.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'framecpp' found
+configure:17040: $? = 1
+configure:17054: result: no
+No package 'framecpp' found
+configure:17070: WARNING: Not found: No package 'framecpp' found
+configure:17126: checking for gds
+configure:17133: $PKG_CONFIG --exists --print-errors "gds >= 2.15.6"
+Package gds was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gds.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gds' found
+configure:17136: $? = 1
+configure:17150: $PKG_CONFIG --exists --print-errors "gds >= 2.15.6"
+Package gds was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gds.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gds' found
+configure:17153: $? = 1
+configure:17167: result: no
+No package 'gds' found
+configure:17184: WARNING: Not found: No package 'gds' found
+configure:17232: checking for LAL
+configure:17239: $PKG_CONFIG --exists --print-errors "lal >= 6.7.0 lalframe >= 1.0.5 lalmetaio >= 1.0.4 lalsupport lalsimulation >= 0.2.0 lalburst >= 1.1.2 lalinspiral >= 1.3.0"
+configure:17242: $? = 0
+configure:17256: $PKG_CONFIG --exists --print-errors "lal >= 6.7.0 lalframe >= 1.0.5 lalmetaio >= 1.0.4 lalsupport lalsimulation >= 0.2.0 lalburst >= 1.1.2 lalinspiral >= 1.3.0"
+configure:17259: $? = 0
+configure:17317: result: yes
+configure:17332: checking for GSTLAL
+configure:17339: $PKG_CONFIG --exists --print-errors "gstlal >= 0.3.0"
+Package gstlal was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gstlal.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gstlal' found
+configure:17342: $? = 1
+configure:17356: $PKG_CONFIG --exists --print-errors "gstlal >= 0.3.0"
+Package gstlal was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gstlal.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gstlal' found
+configure:17359: $? = 1
+configure:17373: result: no
+No package 'gstlal' found
+configure:17389: error: Package requirements (gstlal >= 0.3.0) were not met:
+
+No package 'gstlal' found
+
+Consider adjusting the PKG_CONFIG_PATH environment variable if you
+installed software in a non-standard prefix.
+
+Alternatively, you may set the environment variables GSTLAL_CFLAGS
+and GSTLAL_LIBS to avoid the need to call pkg-config.
+See the pkg-config man page for more details.
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_build=x86_64-pc-linux-gnu
+ac_cv_c_compiler_gnu=yes
+ac_cv_cflags_warn_all=-Wall
+ac_cv_cxx_compiler_gnu=yes
+ac_cv_env_CCC_set=
+ac_cv_env_CCC_value=
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_CXXCPP_set=
+ac_cv_env_CXXCPP_value=
+ac_cv_env_CXXFLAGS_set=set
+ac_cv_env_CXXFLAGS_value='-g -O2'
+ac_cv_env_CXX_set=
+ac_cv_env_CXX_value=
+ac_cv_env_FFTW_CFLAGS_set=
+ac_cv_env_FFTW_CFLAGS_value=
+ac_cv_env_FFTW_LIBS_set=
+ac_cv_env_FFTW_LIBS_value=
+ac_cv_env_GSL_CFLAGS_set=
+ac_cv_env_GSL_CFLAGS_value=
+ac_cv_env_GSL_LIBS_set=
+ac_cv_env_GSL_LIBS_value=
+ac_cv_env_GSTLAL_CFLAGS_set=
+ac_cv_env_GSTLAL_CFLAGS_value=
+ac_cv_env_GSTLAL_LIBS_set=
+ac_cv_env_GSTLAL_LIBS_value=
+ac_cv_env_LAL_CFLAGS_set=
+ac_cv_env_LAL_CFLAGS_value=
+ac_cv_env_LAL_LIBS_set=
+ac_cv_env_LAL_LIBS_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_NDS_CFLAGS_set=
+ac_cv_env_NDS_CFLAGS_value=
+ac_cv_env_NDS_LIBS_set=
+ac_cv_env_NDS_LIBS_value=
+ac_cv_env_ORC_CFLAGS_set=
+ac_cv_env_ORC_CFLAGS_value=
+ac_cv_env_ORC_LIBS_set=
+ac_cv_env_ORC_LIBS_value=
+ac_cv_env_PKG_CONFIG_LIBDIR_set=
+ac_cv_env_PKG_CONFIG_LIBDIR_value=
+ac_cv_env_PKG_CONFIG_PATH_set=set
+ac_cv_env_PKG_CONFIG_PATH_value=/opt/lscsoft/gst/lib/pkgconfig:
+ac_cv_env_PKG_CONFIG_set=
+ac_cv_env_PKG_CONFIG_value=
+ac_cv_env_PYTHON_VERSION_set=
+ac_cv_env_PYTHON_VERSION_value=
+ac_cv_env_PYTHON_set=
+ac_cv_env_PYTHON_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=x86_64-linux-gnu
+ac_cv_env_cairo_CFLAGS_set=
+ac_cv_env_cairo_CFLAGS_value=
+ac_cv_env_cairo_LIBS_set=
+ac_cv_env_cairo_LIBS_value=
+ac_cv_env_framecpp_CFLAGS_set=
+ac_cv_env_framecpp_CFLAGS_value=
+ac_cv_env_framecpp_LIBS_set=
+ac_cv_env_framecpp_LIBS_value=
+ac_cv_env_gds_CFLAGS_set=
+ac_cv_env_gds_CFLAGS_value=
+ac_cv_env_gds_LIBS_set=
+ac_cv_env_gds_LIBS_value=
+ac_cv_env_gstreamer_CFLAGS_set=
+ac_cv_env_gstreamer_CFLAGS_value=
+ac_cv_env_gstreamer_LIBS_set=
+ac_cv_env_gstreamer_LIBS_value=
+ac_cv_env_gstreamer_audio_CFLAGS_set=
+ac_cv_env_gstreamer_audio_CFLAGS_value=
+ac_cv_env_gstreamer_audio_LIBS_set=
+ac_cv_env_gstreamer_audio_LIBS_value=
+ac_cv_env_gstreamer_video_CFLAGS_set=
+ac_cv_env_gstreamer_video_CFLAGS_value=
+ac_cv_env_gstreamer_video_LIBS_set=
+ac_cv_env_gstreamer_video_LIBS_value=
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_pygobject_CFLAGS_set=
+ac_cv_env_pygobject_CFLAGS_value=
+ac_cv_env_pygobject_LIBS_set=
+ac_cv_env_pygobject_LIBS_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_header_dlfcn_h=yes
+ac_cv_header_inttypes_h=yes
+ac_cv_header_memory_h=yes
+ac_cv_header_numpy_arrayobject_h=yes
+ac_cv_header_stdc=yes
+ac_cv_header_stdint_h=yes
+ac_cv_header_stdlib_h=yes
+ac_cv_header_string_h=yes
+ac_cv_header_strings_h=yes
+ac_cv_header_sys_stat_h=yes
+ac_cv_header_sys_types_h=yes
+ac_cv_header_unistd_h=yes
+ac_cv_host=x86_64-pc-linux-gnu
+ac_cv_lib_m_main=yes
+ac_cv_objext=o
+ac_cv_path_EGREP='/bin/grep -E'
+ac_cv_path_FGREP='/bin/grep -F'
+ac_cv_path_GREP=/bin/grep
+ac_cv_path_PYTHON=/usr/bin/python
+ac_cv_path_SED=/bin/sed
+ac_cv_path_ac_pt_PKG_CONFIG=/usr/bin/pkg-config
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_path_mkdir=/bin/mkdir
+ac_cv_prog_AWK=mawk
+ac_cv_prog_CPP='gcc -std=gnu99 -E'
+ac_cv_prog_CXXCPP='g++ -E'
+ac_cv_prog_ac_ct_AR=ar
+ac_cv_prog_ac_ct_CC=gcc
+ac_cv_prog_ac_ct_CXX=g++
+ac_cv_prog_ac_ct_OBJDUMP=objdump
+ac_cv_prog_ac_ct_RANLIB=ranlib
+ac_cv_prog_ac_ct_STRIP=strip
+ac_cv_prog_cc_c89=
+ac_cv_prog_cc_c99=-std=gnu99
+ac_cv_prog_cc_g=yes
+ac_cv_prog_cxx_g=yes
+ac_cv_prog_make_make_set=yes
+am_cv_CC_dependencies_compiler_type=none
+am_cv_CXX_dependencies_compiler_type=none
+am_cv_pathless_PYTHON=python
+am_cv_prog_tar_ustar=gnutar
+am_cv_python_platform=linux2
+am_cv_python_pyexecdir='${exec_prefix}/lib/python2.6/dist-packages'
+am_cv_python_pythondir='${prefix}/lib/python2.6/dist-packages'
+am_cv_python_version=2.6
+lt_cv_ar_at_file=@
+lt_cv_archive_cmds_need_lc=no
+lt_cv_deplibs_check_method=pass_all
+lt_cv_file_magic_cmd='$MAGIC_CMD'
+lt_cv_file_magic_test_file=
+lt_cv_ld_reload_flag=-r
+lt_cv_nm_interface='BSD nm'
+lt_cv_objdir=.libs
+lt_cv_path_LD=/usr/bin/ld
+lt_cv_path_LDCXX='/usr/bin/ld -m elf_x86_64'
+lt_cv_path_NM='/usr/bin/nm -B'
+lt_cv_path_mainfest_tool=no
+lt_cv_prog_compiler_c_o=yes
+lt_cv_prog_compiler_c_o_CXX=yes
+lt_cv_prog_compiler_pic='-fPIC -DPIC'
+lt_cv_prog_compiler_pic_CXX='-fPIC -DPIC'
+lt_cv_prog_compiler_pic_works=yes
+lt_cv_prog_compiler_pic_works_CXX=yes
+lt_cv_prog_compiler_rtti_exceptions=no
+lt_cv_prog_compiler_static_works=yes
+lt_cv_prog_compiler_static_works_CXX=yes
+lt_cv_prog_gnu_ld=yes
+lt_cv_prog_gnu_ldcxx=yes
+lt_cv_sharedlib_from_linklib_cmd='printf %s\n'
+lt_cv_shlibpath_overrides_runpath=no
+lt_cv_sys_global_symbol_pipe='sed -n -e '\''s/^.*[	 ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[	 ][	 ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p'\'' | sed '\''/ __gnu_lto/d'\'''
+lt_cv_sys_global_symbol_to_c_name_address='sed -n -e '\''s/^: \([^ ]*\)[ ]*$/  {\"\1\", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \([^ ]*\)$/  {"\2", (void *) \&\2},/p'\'''
+lt_cv_sys_global_symbol_to_c_name_address_lib_prefix='sed -n -e '\''s/^: \([^ ]*\)[ ]*$/  {\"\1\", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \(lib[^ ]*\)$/  {"\2", (void *) \&\2},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \([^ ]*\)$/  {"lib\2", (void *) \&\2},/p'\'''
+lt_cv_sys_global_symbol_to_cdecl='sed -n -e '\''s/^T .* \(.*\)$/extern int \1();/p'\'' -e '\''s/^[ABCDGIRSTW]* .* \(.*\)$/extern char \1;/p'\'''
+lt_cv_sys_max_cmd_len=3458764513820540925
+lt_cv_to_host_file_cmd=func_convert_file_noop
+lt_cv_to_tool_file_cmd=func_convert_file_noop
+pkg_cv_FFTW_CFLAGS=' '
+pkg_cv_FFTW_LIBS='-lfftw3 -lm  '
+pkg_cv_LAL_CFLAGS=' '
+pkg_cv_LAL_LIBS='-llalframe -lFrame -llalsupport -lz -llalburst -llalinspiral -llalmetaio -llalsimulation -lmetaio -llal -lgsl -lgslcblas -lfftw3 -lfftw3f -lm  '
+pkg_cv_gstreamer_CFLAGS='-pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2  '
+pkg_cv_gstreamer_LIBS='-pthread -L/opt/lscsoft/gst/lib -lgstbase-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lxml2 -lgthread-2.0 -lrt -lglib-2.0  '
+pkg_cv_gstreamer_audio_CFLAGS='-pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2  '
+pkg_cv_gstreamer_audio_LIBS='-pthread -L/opt/lscsoft/gst/lib -lgstaudio-0.10 -lgstbase-0.10 -lgstinterfaces-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lxml2 -lgthread-2.0 -lrt -lglib-2.0  '
+pkg_cv_gstreamer_video_CFLAGS='-pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2  '
+pkg_cv_gstreamer_video_LIBS='-pthread -L/opt/lscsoft/gst/lib -lgstvideo-0.10 -lgstbase-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lxml2 -lgthread-2.0 -lrt -lglib-2.0  '
+pkg_cv_pygobject_CFLAGS='-pthread -I/usr/include/pygtk-2.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  '
+pkg_cv_pygobject_LIBS='-pthread -lgobject-2.0 -lgthread-2.0 -lrt -lglib-2.0  '
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /build/gstlal-ugly/gstlal-ugly-0.2.1/gnuscripts/missing --run aclocal-1.11'
+AMDEPBACKSLASH=''
+AMDEP_FALSE=''
+AMDEP_TRUE='#'
+AMTAR='$${TAR-tar}'
+AM_CFLAGS=' -Wall -Wextra -Wno-missing-field-initializers -Wno-unused-parameter'
+AM_LDFLAGS=' -no-undefined'
+AR='ar'
+AUTOCONF='${SHELL} /build/gstlal-ugly/gstlal-ugly-0.2.1/gnuscripts/missing --run autoconf'
+AUTOHEADER='${SHELL} /build/gstlal-ugly/gstlal-ugly-0.2.1/gnuscripts/missing --run autoheader'
+AUTOMAKE='${SHELL} /build/gstlal-ugly/gstlal-ugly-0.2.1/gnuscripts/missing --run automake-1.11'
+AWK='mawk'
+CC='gcc -std=gnu99'
+CCDEPMODE='depmode=none'
+CFLAGS='-g -O2'
+COND_CAIRO_FALSE=''
+COND_CAIRO_TRUE='#'
+COND_FRAMECPP_FALSE=''
+COND_FRAMECPP_TRUE='#'
+COND_GDS_FALSE=''
+COND_GDS_TRUE='#'
+COND_GSTBASEPARSE_FALSE=''
+COND_GSTBASEPARSE_TRUE='#'
+COND_NDS_FALSE=''
+COND_NDS_TRUE='#'
+CPP='gcc -std=gnu99 -E'
+CPPFLAGS=''
+CXX='g++'
+CXXCPP='g++ -E'
+CXXDEPMODE='depmode=none'
+CXXFLAGS='-g -O2'
+CYGPATH_W='echo'
+DEFS=''
+DEPDIR='.deps'
+DLLTOOL='false'
+DSYMUTIL=''
+DUMPBIN=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP='/bin/grep -E'
+EXEEXT=''
+FFTW_CFLAGS=' '
+FFTW_LIBS='-lfftw3 -lm  '
+FGREP='/bin/grep -F'
+GREP='/bin/grep'
+GSL_CFLAGS=''
+GSL_LIBS=''
+GSTLAL_CFLAGS=''
+GSTLAL_LIBS=''
+GSTLAL_PLUGIN_LDFLAGS='-module -avoid-version -export-symbols-regex _*\(gst_\|Gst\|GST_\|gstlal_\|GstLAL_\|GSTLAL_\).*'
+GSTREAMER_AUDIO_RELEASE='0.10'
+GSTREAMER_RELEASE='0.10'
+GSTREAMER_VIDEO_RELEASE='0.10'
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
+LAL_CFLAGS=' '
+LAL_LIBS='-llalframe -lFrame -llalsupport -lz -llalburst -llalinspiral -llalmetaio -llalsimulation -lmetaio -llal -lgsl -lgslcblas -lfftw3 -lfftw3f -lm  '
+LD='/usr/bin/ld -m elf_x86_64'
+LDFLAGS=''
+LIBAGE='0'
+LIBAPI='1'
+LIBOBJS=''
+LIBREL='0'
+LIBS='-lm '
+LIBTOOL='$(SHELL) $(top_builddir)/libtool'
+LIBVERSION='1:0:0'
+LIPO=''
+LN_S='ln -s'
+LTLIBOBJS=''
+MAKEINFO='${SHELL} /build/gstlal-ugly/gstlal-ugly-0.2.1/gnuscripts/missing --run makeinfo'
+MANIFEST_TOOL=':'
+MIN_GSTREAMER_VERSION='0.10.32'
+MIN_GSTREAMER_VIDEO_VERSION='0.10.32'
+MKDIR_P='/bin/mkdir -p'
+NDS_CFLAGS=''
+NDS_LIBS=''
+NM='/usr/bin/nm -B'
+NMEDIT=''
+OBJDUMP='objdump'
+OBJEXT='o'
+ORC_CFLAGS=''
+ORC_LIBS=''
+OTOOL64=''
+OTOOL=''
+PACKAGE='gstlal-ugly'
+PACKAGE_BUGREPORT='gstlal-discuss@gravity.phys.uwm.edu'
+PACKAGE_NAME='gstlal-ugly'
+PACKAGE_STRING='gstlal-ugly 0.2.1'
+PACKAGE_TARNAME='gstlal-ugly'
+PACKAGE_URL=''
+PACKAGE_VERSION='0.2.1'
+PATH_SEPARATOR=':'
+PKG_CONFIG='/usr/bin/pkg-config'
+PKG_CONFIG_LIBDIR=''
+PKG_CONFIG_PATH='/opt/lscsoft/gst/lib/pkgconfig:'
+PYTHON='/usr/bin/python'
+PYTHON_CPPFLAGS='-I/usr/include/python2.6 -I/usr/lib/pymodules/python2.6/numpy/core/include'
+PYTHON_EXEC_PREFIX='${exec_prefix}'
+PYTHON_EXTRA_LDFLAGS='-Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions'
+PYTHON_EXTRA_LIBS='-lssl -lcrypto  -lssl -lcrypto      -L/usr/lib -lz -lpthread -ldl  -lutil'
+PYTHON_LDFLAGS='-module -avoid-version  -lpython2.6'
+PYTHON_PLATFORM='linux2'
+PYTHON_PREFIX='${prefix}'
+PYTHON_SITE_PKG='/usr/lib/python2.6/dist-packages'
+PYTHON_VERSION='2.6'
+RANLIB='ranlib'
+SED='/bin/sed'
+SET_MAKE=''
+SHELL='/bin/bash'
+STRIP='strip'
+VERSION='0.2.1'
+ac_ct_AR='ar'
+ac_ct_CC='gcc'
+ac_ct_CXX='g++'
+ac_ct_DUMPBIN=''
+am__EXEEXT_FALSE=''
+am__EXEEXT_TRUE=''
+am__fastdepCC_FALSE=''
+am__fastdepCC_TRUE='#'
+am__fastdepCXX_FALSE=''
+am__fastdepCXX_TRUE='#'
+am__include='include'
+am__isrc=''
+am__leading_dot='.'
+am__nodep=''
+am__quote=''
+am__tar='tar --format=ustar -chf - "$$tardir"'
+am__untar='tar -xf -'
+bindir='${exec_prefix}/bin'
+build='x86_64-pc-linux-gnu'
+build_alias='x86_64-linux-gnu'
+build_cpu='x86_64'
+build_os='linux-gnu'
+build_vendor='pc'
+cairo_CFLAGS=''
+cairo_LIBS=''
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datadir}/doc'
+dvidir='${docdir}'
+exec_prefix='NONE'
+framecpp_CFLAGS=''
+framecpp_LIBS=''
+gds_CFLAGS=''
+gds_LIBS=''
+gstreamer_CFLAGS='-pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2  '
+gstreamer_LIBS='-pthread -L/opt/lscsoft/gst/lib -lgstbase-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lxml2 -lgthread-2.0 -lrt -lglib-2.0  '
+gstreamer_audio_CFLAGS='-pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2  '
+gstreamer_audio_LIBS='-pthread -L/opt/lscsoft/gst/lib -lgstaudio-0.10 -lgstbase-0.10 -lgstinterfaces-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lxml2 -lgthread-2.0 -lrt -lglib-2.0  '
+gstreamer_video_CFLAGS='-pthread -I/opt/lscsoft/gst/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2  '
+gstreamer_video_LIBS='-pthread -L/opt/lscsoft/gst/lib -lgstvideo-0.10 -lgstbase-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lxml2 -lgthread-2.0 -lrt -lglib-2.0  '
+host='x86_64-pc-linux-gnu'
+host_alias=''
+host_cpu='x86_64'
+host_os='linux-gnu'
+host_vendor='pc'
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh='${SHELL} /build/gstlal-ugly/gstlal-ugly-0.2.1/gnuscripts/install-sh'
+libdir='${exec_prefix}/lib'
+libexecdir='${prefix}/lib/gstlal-ugly'
+localedir='${datarootdir}/locale'
+localstatedir='/var'
+mandir='${prefix}/share/man'
+mkdir_p='/bin/mkdir -p'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+pkgconfigdir='${libdir}/pkgconfig'
+pkgdocdir='${docdir}/${PACKAGE_NAME}-${PACKAGE_VERSION}'
+pkgpyexecdir='${pyexecdir}/gstlal'
+pkgpythondir='${pythondir}/gstlal'
+plugindir='${exec_prefix}/lib/gstreamer-0.10'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+pyexecdir='${exec_prefix}/lib/python2.6/dist-packages'
+pygobject_CFLAGS='-pthread -I/usr/include/pygtk-2.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  '
+pygobject_LIBS='-pthread -lgobject-2.0 -lgthread-2.0 -lrt -lglib-2.0  '
+pythondir='${prefix}/lib/python2.6/dist-packages'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME "gstlal-ugly"
+#define PACKAGE_TARNAME "gstlal-ugly"
+#define PACKAGE_VERSION "0.2.1"
+#define PACKAGE_STRING "gstlal-ugly 0.2.1"
+#define PACKAGE_BUGREPORT "gstlal-discuss@gravity.phys.uwm.edu"
+#define PACKAGE_URL ""
+#define PACKAGE "gstlal-ugly"
+#define VERSION "0.2.1"
+#define STDC_HEADERS 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_SYS_STAT_H 1
+#define HAVE_STDLIB_H 1
+#define HAVE_STRING_H 1
+#define HAVE_MEMORY_H 1
+#define HAVE_STRINGS_H 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_STDINT_H 1
+#define HAVE_UNISTD_H 1
+#define HAVE_DLFCN_H 1
+#define LT_OBJDIR ".libs/"
+#define HAVE_PYTHON "2.6"
+#define HAVE_LIBM 1
+#define GSTLAL_FFTW_WISDOM_ENV "GSTLAL_FFTW_WISDOM"
+#define HAVE_FRAMECPP test "x${HAVE_FRAMECPP}" == "xyes"
+#define HAVE_GDS test "x${HAVE_GDS}" == "xyes"
+
+configure: exit 1
